src/transport/GroupPeerMessageCounter.cpp

```diff
@@ -238,6 +238,8 @@ void GroupPeerTable::RemoveAndCompactFabric(uint32_t tableIndex)
         return;
     }
     mGroupFabrics[tableIndex].mFabricIndex = kUndefinedFabricIndex;
+    new (&mGroupFabrics[tableIndex]) GroupFabric();
```

Comment:
Is there a way to avoid placement new? Or at least please provide a comment of why placement new is appropriate here (and C++ construction does not work).

Reply:
I'll leave it as it is. why change something that works? Moreover, we are making sure that everything is overwritten without having to define a copy operator.